content/W2015/W15-blog-caleybrock.md

```diff
@@ -14,13 +14,13 @@ Despite her involvement in mathletes and engineering club in high school, Caley
 
 Her passion for design led her to the arts and business program, in which she majored in speech communication with a specialization in digital arts. Dissatisfied with her program, she began taking math courses.
 
-CS 115 (Introduction to Computer Science) was the first time she saw code. "Three years ago, I thought people coded in 0s and 1s", Caley laughs. The course sparked an interest in computer science and made her recall her positive internship experience.
+CS 115 (Introduction to Computer Science) was the first time she saw code. "Three years ago, I thought people coded in 0s and 1s," Caley laughs. The course sparked an interest in computer science and made her recall her positive internship experience.
 
 Rather than just pursuing a minor, she decided to make the full transition into CS at the end of first year.
 
 "I can only go so far with design, but if I build things so many more doors open," she cites.
 
-The transition was a vigourous process, as switching programs is "harder than switching schools". Caley’s high school achievements were not screened. On top of maintaining grades for the arts faculty, she had to take all of the first year CS major courses and maintain at least a 70 average. It took about a year for her to enter the math faculty, become a CS major, and get into the co-op program.
+The transition was a vigourous process, as switching programs is "harder than switching schools". Caley's high school achievements were not screened. On top of maintaining grades for the arts faculty, she had to take all of the first year CS major courses and maintain at least a 70 average. It took about a year for her to enter the math faculty, become a CS major, and get into the co-op program.
```

Comment:
You might want to put that period inside the quotes?

"put into consideration" => "considered"?

"On top of" => "Above and beyond"?